server/src/models/user.model.js

```diff
@@ -0,0 +1,10 @@
+import mongoose from "mongoose";
+
+const userSchema = new mongoose.Schema({
+  email: String,
+  role: String,
+});
+
```

Comment:
The model should include the user's Google ID. `role` should probably include an enum of acceptable values `"user"` and `"admin"`. There should also be a one-to-many relationship between User and Appointment established in both models.

```
const userSchema = new mongoose.Schema({
  googleId: { type: String, required: true },
  email: { type: String, required: true },
  role: { type: String, enum:  ["user", "admin"], required: true },
  appointments: [{ type: mongoose.Schema.Types.ObjectId, ref: 'Appointment' }]
});
```

Reply:
Ahh true, I think I'll change resident to just user as well. and when I was looking at your PR i thought there'd probably be overlap with appointments. Thanks for this!